src/odemis/gui/cont/streams.py

```diff
@@ -331,6 +333,21 @@ def _add_stream_setting_controls(self):
             conf = stream_config.get(vaname)
             self.add_setting_entry(vaname, va, hw_comp=None, conf=conf)
 
+    def _add_axis_controls(self):
+        """
+        Add controls for the axes that are connected to the stream
+        """
+        stream_config = self._stream_config.get(type(self.stream), {})
+        if not hasattr(self.stream, "axis_vas"):
```

Comment:
Please, move this `if` into the caller. It's the same behaviour, but the rest of the functions work like this, so let's follow the flow.